src/test/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerAppointmentValidatorTest.java

```diff
@@ -1827,7 +1827,7 @@ void validateAppointmentDateAfterIncorporationDateWhenValid() {
                 .referenceAppointmentId(FILING_ID)
                 .appointedOn(LocalDate.of(2023, Month.JANUARY, 5))
                 .build();
-        officerAppointmentValidator.validateAppointmentDateBeforeIncorporationDate(request, apiErrorsList, officerFilingDto, companyProfile);
```

Comment:
Are you changing the name back? I think Milan renamed it validateAppointmentDateBeforeIncorporationDate